src/style.css

```diff
@@ -0,0 +1,73 @@
+body {
+  margin: 0;
+}
+
+.snake {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.snake__block {
+  height: 300px;
+  min-width: calc(100%/3);
+  color: white;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  text-align: center;
+  line-height: 3;
+}
+
+.snake--first-box {
+  background-color: rgb(100%,0%,0%);
+}
+
+.snake--second-box {
+  background-color: rgb(80%,0%,0%);
+}
+
+.snake--third-box {
+  background-color: rgb(60%,0%,0%);
+}
+
+.snake--fourth-box {
+  background-color: rgb(40%,0%,0%);
+}
+
+.snake--fifth-box {
+  background-color: rgb(20%,0%,0%);
+}
+
+.snake--sixth-box {
+  background-color: rgb(0%,0%,0%);
+}
+
+@media (min-width: 600px) {
+  .snake {
+    grid-template-columns: repeat(2, 1fr);
+  }
+  .snake--third-box {
+    order: 2;
+  }
+  .snake--fourth-box {
+    order: 3;
+  }
+
+  .snake--fifth-box,
+  .snake--sixth-box {
+    order: 4;
+  }
```

Comment:
why you use order here, you can omit it, since all blocks appear in order of appearance

Reply:
fixed